charts/lfx-v2-project-service/templates/nats-kv-bucket.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+apiVersion: jetstream.nats.io/v1beta2
+kind: KeyValue
+metadata:
+  name: projects
+  namespace: lfx
+spec:
+  bucket: projects
+  history: 20
+  storage: file
+  maxValueSize: 10485760 # 10MB
+  maxBytes: 1073741824 # 1GB
```

Comment:
I'd also suggest making these configurable with defaults (or file a backlog ticket for this)